client/finality-grandpa/src/lib.rs

```diff
@@ -93,10 +93,12 @@ use std::{
 	time::Duration,
 };
 
+const LOG_TARGET: &str = "grandpa";
```

Comment:
This once was "afg". 
My understanding is that AFG was the original GRANDPA name, but maybe is more appropriate to use the actual name

Reply:
I think it is fine. Maybe @andresilva has a different opinion.